engine/apps/api/views/slack_channel.py

```diff
@@ -29,4 +29,4 @@ def get_queryset(self):
             is_archived=False,
         )
 
-        return queryset
+        return queryset.order_by("name")
```

Comment:
Ordering by `id` should be more performant (ordering by indexed vs non-indexed columns). Unless there's a particular reason to order by `name` in this view, I'd suggest to use `id`.
```suggestion
        return queryset.order_by("id")
```